dom-querying/main.js

```diff
@@ -0,0 +1,7 @@
+console.log('hello, world');
+console.log(document.querySelector('h1'));
```

Comment:
Don't forget to label your `console.log`s - especially when you have a lot of them. Should look something like this: 

`console.log('h1', document.querySelector('h1'));`

Reply:
Oh I completely forgot, I will be sure to do that next time. Thank you Kurt!